db/migrate/20200326181705_change_column_genere_to_book.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeColumnGenereToBook < ActiveRecord::Migration[5.1]
+  def change
+    rename_column :books, :genere, :genre
+    rename_column :books, :editor, :publisher
```

Comment:
this does not belong to this PR, change it at the last PR

Reply:
done!